compiler/qsc_parse/src/item.rs

```diff
@@ -561,12 +562,31 @@ fn parse_import_or_export(s: &mut ParserContext) -> Result<ImportOrExportDecl> {
 
 fn parse_import_or_export_item(s: &mut ParserContext) -> Result<ImportOrExportItem> {
     // an import item is a path followed by an optional alias,
-    let path = *(path(s)?);
+    let mut idents = Vec::default();
+    let mut is_glob = false;
+    loop {
+        idents.push(*ident(s)?);
+        if s.peek().kind == TokenKind::Dot {
+            s.advance();
+        }
+        if s.peek().kind == TokenKind::ClosedBinOp(ClosedBinOp::Star) {
+            is_glob = true;
+            s.advance();
+            break;
+        }
+        if s.peek().kind != TokenKind::Ident {
+            break;
+        }
```

Comment:
I think you'll want to avoid breaking here so we loop back around to consume an ident (now that we've seen a dot).

Overall, I think this loop could be cleaned up a bit - maybe just copy the `path` parser and add the check for the `*`?